send-surveys/src/main/java/com/objectcomputing/pulsesurvey/send/surveys/SendSurveysController.java

```diff
@@ -45,34 +51,43 @@ public void setGmailApi(GmailApi api) {
     /* call GetRandomEmails(what percentage of current employees) ->
     GetEmail(Google) -> SelectRandom -> GenerateKeys -> Map<String Email, String KeyUUID>
     Map gets returned */
-    @Override
-    public SendSurveys get() {
+
+    @Post(value = "/")
```

Comment:
Then change this to value="send" so the url will be /surveys/send

Reply:
done